package.json

```diff
@@ -49,11 +53,18 @@
     "utility-types": "^3.10.0"
   },
   "scripts": {
+    "lint": "eslint . --ext .ts",
     "prepare": "husky install",
     "test": "jest --watchAll",
+    "test:ci": "jest --ci",
```

Comment:
Not very familiar with jest, what's the difference between `watchAll` and `ci`?

Reply:
it lets jest know it's running in a ci environment (according to jest [docs](https://jestjs.io/docs/cli#--ci), it looks like it only affects snapshots, which we're not doing here). We're using the same thing in our other projects, all the tests still run